documentation/docs/16-assets.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Asset handling
+---
+
+### Hashing
+
+To include hashes in your asset file names and cache them, you can have Vite process your assets by importing them as shown below:
+
+```html
+<script>
+	import imageSrc from '$lib/assets/image.png';
+</script>
+
+<img src="{imageSrc}" />
```

Comment:
maybe we make this less abstract? also, it should have an alt attribute, and we don't need to enclose it in quotes

```suggestion
<script>
	import logo from '$lib/assets/logo.png';
</script>

<img alt="The project logo" src={logo} />
```